docs/explore_api/templating.md

```diff
@@ -3,307 +3,390 @@
-change those simulation configuration files so that we can run 50000 *different*
-simulations, each one being a slight variation of the other?
-
-## Templating
-
-This is where Inductiva’s templating mechanism comes into play. Templating allows
-you to start with a specific simulation file – your “base case” – containing fixed
-values for the parameters you wish to explore and transform those fixed values into
-variables that you can now change programmatically from your Python code before you
-submit the simulation for remote execution. Let's illustrate the power of templating
-in a simple simulation case, from which you will be able to generalize to your
-own cases.
-
-## Experimenting fluids with different properties
-
-Suppose you want to study how fluids with different properties fall inside a container
-with a cubic shape. More specifically, you want to study the effect of fluid density
-and kinematic viscosity in the splashing on the walls of the cubic container. For that,
-you wish to experiment with 4 values of fluid density and 4 values of kinematic
-viscosity. Overall, you will need to run 16 simulations. 
-
-For the simulation of this scenario, you choose the SplishSplash simulator, and you
-start by configuring your base case with a water fluid. The corresponding simulation files
-are a `unit_box.obj` that defines the domain and the fluid block and the following
-_json_ file (we designate by `config.json`):
-
-```json
-{
-    "Configuration": {
-        "stopAt": 1,
-        "timeStepSize": 0.01,
-        "particleRadius": 0.01,
-        "simulationMethod": 4,
-        "boundaryHandlingMethod": 0,
-        "kernel": 1,
-        "cflMethod": 1,
-        "cflFactor": 0.5,
-        "cflMinTimeStepSize": 0.0001,
-        "cflMaxTimeStepSize": 0.005,
-        "gravitation": [0, 0, -9.81],
-        "gradKernel": 1,
-        "enableVTKExport": true,
-        "dataExportFPS": 60,
-        "particleAttributes": "velocity;density"
-    },
-    "RigidBodies": [
-        {
-            "geometryFile": "unit_box.obj",
-            "translation": [0, 0, 0],
-            "scale": [1, 1, 1],
-            "isDynamic": false
-        }
-    ],
-    "Materials": [
-        {
-            "id": "Fluid",
-            "density0": 1000,
-            "viscosity": 1e-6,
-            "viscosityMethod": 6
-        }
-    ],
-    "FluidModels": [
-        {
-            "id": "Fluid",
-            "particleFile": "unit_box.obj",
-            "translation": [0, 0, 0],
-            "scale": [0.5, 0.5, 0.5],
-            "initialVelocity": [0, 0, 0]
-        }
-    ]
-}
+what you need when you want to find the optimal solution for a problem, and the
+only way to test each candidate's solution is by simulating it. To do so, one needs
+to configure multiple simulations, each with a slightly different set of parameters.
+Generally speaking, the goal is to explore the largest possible volume of the
```

Comment:
Volume here seems weird, and in fact, the goal is to explore the design space. The solution refers to results of the simulation you are obtaining.